website/static/js/citationGrid.js

```diff
@@ -378,10 +368,12 @@ CitationGrid.prototype.makeBibliography = function(folder, format) {
 };
 
 CitationGrid.prototype.getBibliography = function(folder, format) {
-    if (format) {
-        return this.makeBibliography(folder, format);
+
+    if(this.bibliographies[folder.id]){
```

Comment:
Can we make this block a smidge cleaner by assigning `this.bibliographies[folder.id]` to a variable?

Reply:
I was able to clean this up, but we can't really assign this as a variable productively because it's a we still we to reassign the value within the dict.